hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -0,0 +1,941 @@
+      pageSizeOptions: ['10', '20', '30', '50'],
+      showQuickJumper: true,
+      showSizeChanger: true,
+      onShowSizeChange: this.onShowSizeChange,
+      //current:this.state.currentPage,
+      //onChange: this.onChangePagination,
+      itemRender: this.itemRender
+    };
+
+    const generateTable = (dataSource: any) => {
+      return <Table
+        expandRowByClick dataSource={dataSource}
+        columns={this.searchColumn()}
+        loading={loading}
+        pagination={paginationConfig} rowKey='containerId'
+        expandedRowRender={this.expandedRowRender} onExpand={this.onRowExpandClick} />
+    }
+
+    const generateKeyTable = (dataSource: any) => {
+      return <Table
+        expandRowByClick dataSource={dataSource}
+        columns={this.searchKeyColumn()}
+        loading={loading}
+        pagination={paginationConfig} />
+    }
+
+    const generatePendingTable = (dataSource: any) => {
+
+      return <Table
+        expandRowByClick dataSource={dataSource}
+        columns={this.searchPendingColumn()}
+        loading={loading}
+        pagination={paginationConfig} rowKey='keyName' />
+    }
+
+    const generateDeletedTable = (dataSource: any) => {
+      return <Table
+        expandRowByClick dataSource={dataSource}
+        columns={this.searchscmKeyColumn()}
+        loading={loading}
+        pagination={paginationConfig} rowKey='containerId'
+        expandedRowRender={this.expandedRowRender} onExpand={this.onRowExpandClick}
+      />
+    }
+
+    return (
+      <div className='missing-containers-container'>
+        <div className='page-header'>
+          OM DB Insights
+        </div>
+        <div className='content-div'>
+          <Tabs defaultActiveKey='1' onChange={this.changeTab}>
+            <TabPane key='1' tab={`Container Mismatch Info${(mismatchDataSource && mismatchDataSource.length > 0) ? ` (${mismatchDataSource.length})` : ''}`}>
+              {generateTable(mismatchDataSource)}
+            </TabPane>
+            <TabPane key='2' tab={`Open Keys${(openKeysDataSource && openKeysDataSource.length > 0) ? ` (${openKeysDataSource.length})` : ''}`}>
+              {generateKeyTable(openKeysDataSource)}
+            </TabPane>
+            <TabPane key='3'
+              tab={<label>Keys Pending for Deletion{(pendingDeleteKeyDataSource && pendingDeleteKeyDataSource.length > 0) ? ` (${pendingDeleteKeyDataSource.length})` : ''}
+                <Tooltip placement='top' title="Keys and Directories that are pending for deletion">
+                  <Icon type='info-circle' />
+                </Tooltip>
+              </label>
+              }>
+              {generatePendingTable(pendingDeleteKeyDataSource)}
+            </TabPane>
+            <TabPane key='4'
+              tab={<label>Deleted Container Keys {(scmKeyDataSource && scmKeyDataSource.length > 0) ? ` (${scmKeyDataSource.length})` : ''}
+                <Tooltip placement='top' title="Containers in DELETED state in SCM ">
```

Comment:
Padding is not here as well

Reply:
@devabhishekpal 
Completed Change in latest commit.

Reply:
@devabhishekpal 
Completed Change in latest commit.